src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -32,7 +32,13 @@ static XmlSerializerTests()
             method.Invoke(null, new object[] { 1 });
 #endif
 #if XMLSERIALIZERGENERATORTESTS
-            method.Invoke(null, new object[] { 3 });
+            string path = Path.GetDirectoryName(typeof(TypeWithDateTimeStringProperty).Assembly.Location);
```

Comment:
Nit: it might be better to use `typeof(XmlSerializerTests)` other than a random type.